src/uct/cuda/cuda_iface.c

```diff
@@ -68,7 +68,6 @@ static ucs_status_t uct_cuda_iface_query(uct_iface_h iface,
 static uct_iface_ops_t uct_cuda_iface_ops = {
     .iface_close         = UCS_CLASS_DELETE_FUNC_NAME(uct_cuda_iface_t),
     .iface_get_address   = uct_cuda_iface_get_address,
-    .iface_flush         = (void*)ucs_empty_function_return_success,
```

Comment:
empty flush function in tl_base.c is used instead. The function increaments flush counter